rxandroidble/src/main/java/com/polidea/rxandroidble/internal/RxBleRadioOperation.java

```diff
@@ -49,13 +49,13 @@ public void call(Emitter<T> emitter) {
     }
 
     /**
-     * This method will be overridden in a concrete operation implementations and will contain specific operation logic.
+     * This method should be overridden in a concrete operation implementations and should contain specific operation logic.
      *
      * Implementations should call emitter methods to inform the outside world about emissions of `onNext()`/`onError()`/`onCompleted()`.
      * Implementations must call {@link RadioReleaseInterface#release()} at appropriate point to release the radio for any other operations
      * that are queued.
      *
-     * If the emitter has been canceled it is response of the operation to call {@link RadioReleaseInterface#release()} when possible
+     * If the emitter is cancelled it is a responsibility of the operation to call {@link RadioReleaseInterface#release()}, when possible
```

Comment:
Maybe ` If the emitter is cancelled, a responsibility of the operation is to call` or ` If the emitter is cancelled the operation is obligated to call `?